### PR TITLE
KEYCLOAK-18113 Refactor some executor/condition provider IDs

### PR DESCRIPTION
**Pull request by mposolda**

### CODE REVIEW

**Comment by mposolda** [comment]
@vmuzikar @tnorimat Do you please have a chance to review this? See JIRA description for more details https://issues.redhat.com/browse/KEYCLOAK-18113

**Review by tnorimat — DISMISSED** [review]

**Comment by tnorimat** [comment]
@mposolda LGTM.

**Review by vmuzikar — DISMISSED** [review]
LGTM, added two suggestion, just nitpicks. Fix it only if you want. ;)

File: services/src/main/java/org/keycloak/services/clientpolicy/executor/PKCEEnforceExecutorFactory.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
  */
 public class PKCEEnforceExecutorFactory implements ClientPolicyExecutorProviderFactory  {
 
-    public static final String PROVIDER_ID = "pkce-enforce-executor";
+    public static final String PROVIDER_ID = "pkce-enforce";
```
[the remark this answers is not in the document]
Comment (mposolda) [reply services/src/main/java/org/keycloak/services/clientpolicy/executor/PKCEEnforceExecutorFactory.java]: I've changed to `pkce-enforcer` . Thanks

File: services/src/main/java/org/keycloak/services/clientpolicy/executor/HolderOfKeyEnforceExecutorFactory.java [hunk 2]
```diff
@@ -28,7 +28,7 @@
 
 public class HolderOfKeyEnforceExecutorFactory implements ClientPolicyExecutorProviderFactory {
 
-    public static final String PROVIDER_ID = "holder-of-key-enforce-executor";
+    public static final String PROVIDER_ID = "holder-of-key-enforce";
```
[the remark this answers is not in the document]
Comment (mposolda) [reply services/src/main/java/org/keycloak/services/clientpolicy/executor/HolderOfKeyEnforceExecutorFactory.java]: I've changed to `holder-of-key-enforcer` . Thanks

**Comment by mposolda** [comment]
@vmuzikar Thanks for the feedback. I've updated as you suggested

**Review by vmuzikar — APPROVED** [review]
@mposolda Thanks!

**Comment by mposolda** [comment]
@vmuzikar Thanks to you as well!

**Pull request merged**